### ISSUE TITLE
[skyrl-train] Add ScaleRL recipe

### ISSUE THREAD

**Issue by erictang000** [NovaSky-AI/SkyRL#495]
- [x] Add CISPO policy loss (#523)
- [ ] Update dynamic sampling to allow dropping 0 variance samples (right now we either resample or replace)
- [ ] Add adaptive prompt filtering (removing easy prompts across epochs)
- [ ] fp32 logit head
- [ ] Prompt level loss aggregation (vs. token level) for DAPO

**Comment by vutrung96** [comment]
would love to have this
